s4e-backend/src/main/java/pl/cyfronet/s4e/listener/ExpertHelpListener.java

```diff
@@ -0,0 +1,68 @@
+package pl.cyfronet.s4e.listener;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.context.MessageSource;
+import org.springframework.context.event.EventListener;
+import org.springframework.scheduling.annotation.Async;
+import org.springframework.stereotype.Component;
+import org.springframework.transaction.annotation.Transactional;
+import org.thymeleaf.TemplateEngine;
+import org.thymeleaf.context.Context;
+import pl.cyfronet.s4e.event.OnSendHelpRequestEvent;
+import pl.cyfronet.s4e.service.InstitutionService;
+import pl.cyfronet.s4e.service.MailService;
+import pl.cyfronet.s4e.util.MailHelper;
+
+@Component
+@RequiredArgsConstructor
+public class ExpertHelpListener {
+    public final static String expertMail = "xyz@mail.pl";
```

Comment:
I'd externalize this somewhere. Either to system property or the property DB table